lib/backends/sdc/utils.js

```diff
@@ -110,45 +110,46 @@ function getPublishingRules(opts, fwrules) {
  *
  * In 1.9.1:
  *
- *     filters={"label":["com.joyent.package=sample-2G"]}
+ *     filters={"label":["com.joyent.package=sample-2G", "foo=bar"]}
  *
  * In 1.10.0:
  *
- *     filters={"label":{"com.joyent.package=sample-2G":true}}
- *
- * This function takes the filters[label] value (as the filters parameter here)
- * and returns a normalized array of the packages we want to filter on.
+ *     filters={"label":{"com.joyent.package=sample-2G":true, "foo=bar":true}}
  *
+ * This function takes the filters object and returns a new normalized object
+ * with each object value being an array instance.
```

Comment:
I.e. the normalized form is  what it looks like in 1.9.1, right?  Might be clearer to state that or just say "the former".